lib/puppet/provider/gluster_volume/gluster_volume.rb

```diff
@@ -4,7 +4,7 @@
   :gluster_volume, :parent => Puppet::Provider::Gluster) do
 
   has_command(:gluster, 'gluster') do
-    environment :HOME => "/tmp"
+    environment :HOME => '/tmp'
```

Comment:
Can you do a grep for `"` across all the source so we can get all the `"` -> `'` once and for all?